src/common/options/global.yaml.in

```diff
@@ -3645,6 +3645,12 @@ options:
   desc: Cost for scrub operations in work queue
   default: 50_M
   with_legacy: true
+- name: osd_scrub_event_cost
```

Comment:
Given that this is a temporary solution, I don't think the config option is required. The only reason to keep it is if we want a way to change the scrub cost in case the default 4K doesn't work well. I would go with what others think is best. If we do decide to go with it, I would prefer to call it 'osd_mclock_scrub_cost'.

Reply:
I think it's worth adding the config.  We might need to experiment with it, and it's just kind of always helpful to be able to modify these things at runtime.

Reply:
Sounds fine to me.